packages/analytics/src/integrations/Omnitracking/definitions.ts

```diff
@@ -664,7 +664,11 @@ export const trackEventsMapper: Readonly<OmnitrackingTrackEventsMapper> = {
     filtersApplied: getFilterParametersFromEvent(data.properties),
   }),
   [EventType.InteractContent]: (data: EventData<TrackTypesValues>) => {
-    if (data.properties?.interactionType === InteractionType.Scroll) {
+    if (
+      data.properties?.interactionType === InteractionType.Scroll &&
+      data.properties.contentType !== 'carousel' &&
```

Comment:
Could we add new contentType/elementType enum type (like integrationType), replacing this fixed values?

Reply:
Fixed!